packages/indexer-cli/src/commands/indexer/status.ts

```diff
@@ -136,15 +136,16 @@ module.exports = {
       const keys = Object.keys(pickFields(result.data.indexerEndpoints, []))
       keys.sort()
 
+      const status_up = outputFormat == 'table' ? chalk.green('up') : 'up'
+      const status_down = outputFormat == 'table' ? chalk.red('down') : 'down'
```

Comment:
nit: JS variables typically use `camelCase`, so I'd rename these to `statusUp` and `statusDown`.

Reply:
Fixed